contracts/modules/ModuleFactory.sol

```diff
@@ -94,10 +95,17 @@ contract ModuleFactory is IModuleFactory, Ownable {
     }
 
     function _changeUsageCost(uint256 _usageCost) internal {
-        require(now.sub(lastTimeUsageCostChange) > LOCKPERIOD, "Under lock period");
         lastTimeUsageCostChange = now;
-        emit ChangeUsageCost(usageCost, _usageCost);
-        usageCost = _usageCost;
+        emit ChangeUsageCost(usageFee, _usageCost);
+        tempUsageFee = usageFee;
+        usageFee = _usageCost;
+    }
+
+    function usageCost() public view returns(uint256 usdUsageCost) {
+        if (now > lastTimeUsageCostChange.add(COLDPERIOD))
+            return usageFee;
+        else
+            return tempUsageFee;
```

Comment:
This can be bypassed by changing the fee twice. We should modify the change usage cost function to only update the tempUsageFee if `now > lastTimeUsageCostChange.add(COLDPERIOD)`.

Reply:
Yeah - I tend to think of this as needing two operations from the factory owner:
1. set proposed new usage fee (emit UsageFeeProposed(proposedFee, currentFee))
2. confirm proposed usage fee (can only be called more than 24 hours after 1 above) (emit ChangeUsageCost(oldFee, newFee))

This means that if 2. never happens, the fee is never changed. I see this as something that would happen rarely, so not an issue that there needs to be two calls.